src/expertsystem/reaction/topology.py

```diff
@@ -476,17 +576,6 @@ def build_graphs(
 
                 extendable_graph_list.extend(self.extend_graph(active_graph))
 
-            # check if two topologies are the same
```

Comment:
Why has this become redundant?

Reply:
Well the check was never performed since is isomorphic function is not implemented. Hence I just deleted unused code.